enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/BackButton.tsx

```diff
@@ -0,0 +1,10 @@
+import { useInteractiveQuestionContext } from "embedding-sdk/components/public/InteractiveQuestion/context";
+import { DashboardBackButton } from "metabase/query_builder/components/view/ViewHeader/components";
+
+export const QuestionBackButton = () => {
```

Comment:
nit: I'm not sure if this is ever discussed, but I rarely see file names that are different from the exported main components. I spotted this problem because I saw `QuestionBackButton` and tried to find this file, but it was named differently.

This should help with searching for components as well if they match, could they be the same?

Reply:
Weird. I never do this exactly because it's super hard to find components. Not sure what happened there. I'll change it